libs/npm/core/src/components/navbar/NavBarItem.tsx

```diff
@@ -5,17 +5,21 @@ import * as styled from './styled';
 import { isActive } from './utils';
 
 export interface INavBarItem {
+  /** The unique key to identify this item. */
+  uid?: string;
   /**
    * choose the tab label
    */
-  label?: string;
+  label?: React.ReactNode;
```

Comment:
Not important for this PR, but in the future we can add more advanced navbar items.